File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.cpp
class Solution {
public:
    int longestString(int x, int y, int z) {
        int ans = z;
        ans +=(min(x,y));
        if(max(x,y)==min(x,y)){
            ans+=min(x,y);
        }else{
            ans+=min(x,y)+1;
        }
        return ans+ans;
    }
};

File: 2745-construct-the-longest-new-string/2745-construct-the-longest-new-string.py
class Solution:
    def longestString(self, x: int, y: int, z: int) -> int:
        ans = z
        if(max(x,y)==min(x,y)):
            ans +=(2*min(x,y))
        else:
            ans +=1
            ans +=(2*min(x,y))
        return ans*2